web-ng/src/app/components/map/ground-pin.ts

```diff
@@ -14,8 +14,14 @@
  * limitations under the License.
  */
 
-export const renderPin = (color: string) => {
-  return svgTemplate.replace('{{ color }}', color);
+export const defaultIconColor = 'red';
+
+export const renderPin = (color: string | undefined) => {
+  return svgTemplate.replace('{{ color }}', color || defaultIconColor);
+};
+
+export const getPinSrc = (color: string | undefined) => {
+  return '/assets/img/pin-red.svg';
```

Comment:
Can you we renderPin() here instead of having a static marker?

Reply:
done